src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -19,6 +19,77 @@ See License.txt for details.
+const char* vtkPlusWinProbeVideoSource::SET_FOCAL_DEPTH              = "SetFocalDepth";
+const char* vtkPlusWinProbeVideoSource::GET_FOCAL_DEPTH              = "GetFocalDepth";
+const char* vtkPlusWinProbeVideoSource::SET_ALL_ARFI_FOCAL_DEPTHS    = "SetAllARFIFocalDepths";
+const char* vtkPlusWinProbeVideoSource::SET_ARFI_FOCAL_DEPTHS        = "SetARFIFocalDepth";
+const char* vtkPlusWinProbeVideoSource::GET_ARFI_FOCAL_DEPTHS        = "GetARFIFocalDepth";
+const char* vtkPlusWinProbeVideoSource::SET_B_MULTIFOCAL_ZONE_COUNT  = "SetBMultifocalZoneCount";
+const char* vtkPlusWinProbeVideoSource::GET_B_MULTIFOCAL_ZONE_COUNT  = "GetBMultifocalZoneCount";
+const char* vtkPlusWinProbeVideoSource::SET_FIRST_GAIN_VALUE         = "SetFirstGainValue";
+const char* vtkPlusWinProbeVideoSource::GET_FIRST_GAIN_VALUE         = "GetFirstGainValue";
+const char* vtkPlusWinProbeVideoSource::SET_TGC_OVERALL_GAIN         = "SetTGCOverallGain";
+const char* vtkPlusWinProbeVideoSource::GET_TGC_OVERALL_GAIN         = "GetTGCOverallGain";
+const char* vtkPlusWinProbeVideoSource::SET_SPATIAL_COMPOUND_ENABLED = "SetSpatialCompoundEnabled";
+const char* vtkPlusWinProbeVideoSource::GET_SPATIAL_COMPOUND_ENABLED = "GetSpatialCompoundEnabled";
+const char* vtkPlusWinProbeVideoSource::GET_SPATIAL_COMPOUND_ANGLE   = "GetSpatialCompoundAngle";
+const char* vtkPlusWinProbeVideoSource::SET_SPATIAL_COMPOUND_COUNT   = "SetSpatialCompoundCount";
+const char* vtkPlusWinProbeVideoSource::GET_SPATIAL_COMPOUND_COUNT   = "GetSpatialCompoundCount";
+const char* vtkPlusWinProbeVideoSource::SET_MMODE_ENABLED            = "SetMModeEnabled";
+const char* vtkPlusWinProbeVideoSource::GET_MMODE_ENABLED            = "GetMModeEnabled";
+const char* vtkPlusWinProbeVideoSource::SET_M_REVOLVING_ENABLED      = "SetMRevolvingEnabled";
+const char* vtkPlusWinProbeVideoSource::SET_RF_MODE_ENABLED          = "SetRfModeEnabled";
+const char* vtkPlusWinProbeVideoSource::GET_RF_MODE_ENABLED          = "GetRfModeEnabled";
+const char* vtkPlusWinProbeVideoSource::SET_MPR_FREQUENCY            = "SetMPRFrequency";
+const char* vtkPlusWinProbeVideoSource::GET_MPR_FREQUENCY            = "GetMPRFrequency";
+const char* vtkPlusWinProbeVideoSource::SET_M_LINE_INDEX             = "SetMLineIndex";
+const char* vtkPlusWinProbeVideoSource::GET_M_LINE_INDEX             = "GetMLineIndex";
+const char* vtkPlusWinProbeVideoSource::SET_M_LINE_COUNT             = "SetMLineCount";
+const char* vtkPlusWinProbeVideoSource::GET_M_LINE_COUNT             = "GetMLineCount";
+const char* vtkPlusWinProbeVideoSource::SET_M_WIDTH                  = "SetMWidth";
+const char* vtkPlusWinProbeVideoSource::GET_M_WIDTH                  = "GetMWidth";
+const char* vtkPlusWinProbeVideoSource::SET_M_DEPTH                  = "SetMDepth";
+const char* vtkPlusWinProbeVideoSource::GET_M_DEPTH                  = "GetMDepth";
+const char* vtkPlusWinProbeVideoSource::SET_DECIMATION               = "SetDecimation";
```

Comment:
Looks like a missed command for `GetDecimation` which would call to the `GetSSDecimation` method in this class. You could also set the command to be the same as the method name in the class. The commands would be `SetSSDecimation`/`GetSSDecimation`

Reply:
updated